_data/toc.yaml

```diff
@@ -157,6 +155,10 @@ guides:
       title: Manage data in containers
   - path: /engine/docker-overview/
     title: Docker overview
+  - path: /release-notes/docker-ce/
```

Comment:
Because some of them are already in other places, I was trying to replicate the "other places" idea, but we don't have a "Docker Engine" specific area anymore. I need to put these stubs somewhere or they won't have a left-hand nav at all.